packages/app/nuxt.config.js

```diff
@@ -19,16 +19,29 @@ export default {
   ],
   buildModules: [
     '@nuxtjs/pwa',
-    '@nuxtjs/google-analytics'
   ],
-  googleAnalytics: {
-    id: process.env.GA
-  },
   modules: [
     '@nuxtjs/axios',
     '@nuxtjs/pwa',
-    'nuxt-facebook-pixel-module'
+    'nuxt-facebook-pixel-module',
+    '@nuxtjs/google-gtag'
   ],
+  'google-gtag': {
+    id: 'G-RXT270TF63', // required
```

Comment:
Why not to use environment variable (e.g. `process.env.GOOGLE_GTAG`) instead of hardcoded value?